src/main/types/SIOP.types.ts

```diff
@@ -452,30 +452,30 @@ export enum VerificationMode {
   EXTERNAL,
 }
 
-export interface InternalVerification {
+export interface Verification {
+  checkLinkedDomain?: CheckLinkedDomain;
   mode: VerificationMode;
-  /*registry?: string;
-        rpcUrl?: string;*/
   resolveOpts: ResolveOpts;
 }
 
-export interface ExternalVerification {
-  mode: VerificationMode;
+export type InternalVerification = Verification;
+
+export interface ExternalVerification extends Verification {
   verifyUri: string; // url to call to verify the id_token signature
   authZToken?: string; // Optional: bearer token to use to the call
   resolveOpts: ResolveOpts;
 }
 
 export interface VerifyAuthenticationRequestOpts {
-  verification: InternalVerification | ExternalVerification; // To use internal verification or external hosted verification
+  verification: Verification; // To use internal verification or external hosted verification
   checkLinkedDomain?: CheckLinkedDomain;
   // didDocument?: DIDDocument; // If not provided the DID document will be resolved from the request
   nonce?: string; // If provided the nonce in the request needs to match
   // redirectUri?: string;
 }
 
 export interface VerifyAuthenticationResponseOpts {
-  verification: InternalVerification | ExternalVerification;
+  verification: Verification;
```

Comment:
Don't change this to the super interface. Right now this interface is exactly the same as the InternalVerification. But if we add one unique prop to InternalVerification you would still be able to pass in objects with to few params, as long as they have all props from Verification. It really needs to be either Internal or External Verification

Reply:
reverted to `InternalVerification | ExternalVerification`